sci-biology/foldingathome/foldingathome-8.4.5.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..13} )
+inherit python-any-r1 scons-utils systemd
+
+DESCRIPTION="Distributed computing project for simulating protein dynamics."
+HOMEPAGE="https://foldingathome.org"
+SRC_URI="
+	https://github.com/FoldingAtHome/fah-client-bastet/archive/refs/tags/v${PV}.tar.gz
+		-> foldingathome-client-v${PV}.tar.gz
+	https://github.com/CauldronDevelopmentLLC/cbang/archive/refs/tags/bastet-v${PV}.tar.gz
+		-> foldingathome-cbang-v${PV}.tar.gz
+"
+
+S="${WORKDIR}/fah-client-bastet-${PV}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="video_cards_amdgpu"
+
+DEPEND="
+	acct-group/foldingathome
+	acct-user/foldingathome
+	app-arch/bzip2
+	app-arch/lz4
+	dev-db/sqlite:3
+	dev-libs/expat
+	dev-libs/openssl
+	sys-apps/systemd
+	sys-libs/zlib
+	video_cards_amdgpu? ( dev-libs/rocm-opencl-runtime )
```

Comment:
Why is this USE-conditional dependency in DEPEND? This indicates it is needed for compiling, but no scons options are used to enable this. If I grep around in the code, cbang-bastet-v8.4.5/src/cbang/hw/CUDALibrary.cpp indicates that it checks for e.g. libcuda.so at runtime by attempting to dlopen it.

Same with cbang-bastet-v8.4.5/src/cbang/hw/OpenCLLibrary.cpp  for libOpenCL.so (dev-libs/opencl-icd-loader).

If it is not needed at build time, only at runtime, then this line should be moved to RDEPEND.

Reply:
done